documentation/wtpdf-from-latex.md

```diff
@@ -32,7 +32,7 @@ so that it is not necessary to specify individual modules.
   lang        = en,
   pdfversion  = 2.0,
   pdfstandard = ua-2,
-  pdfversion  = 2.0,  % or 1.7  if desired pdfstandard = ua-2
+  pdfversion  = 2.0,  % or 1.7 if desired pdfstandard = ua-1
```

Comment:
diff looks `pdfversion =` twice, some problem of the merge, or do I see ghosts?

Reply:
sigh, sorry I'll clean up. Not sure if the PR was wrong or my hand merge messed up but either way it's wrong now.